libs/utils/src/file_reading.ts

```diff
@@ -0,0 +1,73 @@
+import JsZip, { JSZipObject } from 'jszip';
+import { Buffer } from 'buffer';
+
+export function readFileAsyncAsString(file: File): Promise<string> {
+  return new Promise<string>((resolve, reject) => {
+    const reader = new FileReader();
+    reader.onload = () => {
+      const { result } = reader;
+      resolve(typeof result === 'string' ? result : '');
+    };
+    reader.onerror = reject;
```

Comment:
I know this is just an existing function that was moved but should this `onerror` look like the one for `readFile` below?

```ts
reader.onerror = () => {
  reject(reader.error);
};
```